specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+      }
+    },
+    "/{idScope}/devices/{registrationId}/certificates/issue": {
+      "post": {
+        "tags": [
+          "POST"
+        ],
+        "summary": "API to issue certificates for IoT devices.",
+        "operationId": "CertificateIssuance_IssueCertificate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "idScope",
+            "in": "path",
+            "description": "The service assigned ID scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registrationId",
+            "in": "path",
+            "description": "The device registration ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "request",
+            "in": "body",
+            "description": "Certificate signing request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateIssuanceRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceOperationStatusResponse"
+            },
+            "headers": {
+              "location": {
+                "description": "The location of the resource.",
+                "type": "string"
+              },
+              "retry-after": {
+                "description": "The retry interval in seconds.",
+                "type": "integer",
+                "format": "int32"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceErrorDetails"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Request for issuing a certificate with a certificate signing request": {
+            "$ref": "./examples/device/CertificateIssuance/issueCertificate.json"
+          }
+        }
+      }
+    },
+    "/{idScope}/registrations/{registrationId}": {
+      "post": {
+        "tags": [
+          "POST",
+          "RuntimeRegistration"
+        ],
+        "summary": "Gets the device registration status.",
```

Comment:
Why a POST to *get* device registration status? If this is a status monitor as part of an LRO, it should support a GET.